tools/clang/tools/dxa/dxa.cpp

```diff
@@ -546,7 +546,7 @@ int main(int argc, const char **argv) {
       return 2;
     }
 
-    DxcDllSupport dxcSupport;
+    LibraryDllLoader dxcSupport;
     dxc::EnsureEnabled(dxcSupport);
```

Comment:
Should this (and the member) be a `DxCompilerDllLoader`? EnsureEnabled isn't doing much here since this is only ever used for dxcompiler.dll
```suggestion
    DxCompilerDllLoader dxcSupport;
    IFT(dxcSupport.Initialize());
```

Reply:
Yeah, there is no setup function, so this is pretty clearly only DxCompilerDllLoader.